examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
If this is a new file, then you should set the copyright date to 2018.  If you just renamed it, then leave it as-is (but add the deleted file to the PR so that git can see it as a rename).

Reply:
I renamed cupertino_dialog_demo to be cupertino_alert_demo - is that okay, or should I leave the file name as is?

Reply:
It's OK to rename it, you just had to add the deleted file. It also affects what gets shown in the diffs: if you rename it, the new file will just show the diffs from the old file, but if you don't delete the old one, then it thinks it's a new file.

Reply:
Ah okay, that makes sense